sql/timeseries/asyncClipboardRead.sql

```diff
@@ -0,0 +1,23 @@
+#standardSQL
+SELECT
+  REGEXP_REPLACE(yyyymmdd, r"(\d{4})(\d{2})(\d{2})", "\\1_\\2_\\3") AS date,
+  UNIX_DATE(CAST(REGEXP_REPLACE(yyyymmdd, r"(\d{4})(\d{2})(\d{2})", "\\1-\\2-\\3") AS DATE)) * 1000 * 60 * 60 * 24 AS timestamp,
+  client,
+  num_urls,
+  ROUND(num_urls / total_urls * 100, 5) AS percent,
+  ANY_VALUE(sample_urls) sample_urls,
```

Comment:
If `sample_urls` would be useful, could we link the metric description to the corresponding Chrome Status dashboard? Otherwise I don't think the reporting UI can support an array of URLs.

Reply:
We certainly could. The URL structure is https://chromestatus.com/metrics/feature/timeline/popularity/${featureID}. Should I remove `sample_urls` from the queries again?

Reply:
Yes please 😅 

`num_urls` is a simple value we can show in the table so there's no problem keeping that.

Reply:
Done ✅

Reply:
Great thanks!